Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG CLI_VERSION=latest
+FROM microsoft/powershell:latest
+ARG CLI_VERSION=latest
+RUN apt-get update && apt-get install -y curl sudo
+RUN curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+RUN apt install nodejs -y
+RUN npm i -g @pnp/cli-microsoft365@${CLI_VERSION} --production
+RUN pwsh -c 'm365 cli completion pwsh setup --profile /root/.config/powershell/Microsoft.PowerShell_profile.ps1'
```

Comment:
Shall we also setup completion for *sh?

Reply:
This one needs some further investigation. 

When executing the `setup` command for `sh`, an error is thrown, `Shell cannot be detected` which I assume is being thrown from `omelette`.

`echo $0` returns `bash` when executed within the container, so not sure on why it can't be detected by `omelette`.

Reply:
Could it be that it's trying to detect the shell by looking at the SHELL env var?

Reply:
Checking the env vars, both are set...

![image](https://user-images.githubusercontent.com/11563347/98012888-858f0a80-1df1-11eb-973f-4aa42ca99d34.png)

Reply:
Further investigation, `omelette` uses `process.env.SHELL` to determine the shell, this is not set in the container and therefore throws the error.

![image](https://user-images.githubusercontent.com/11563347/98032563-183ca300-1e0c-11eb-9a92-accb2be0ea32.png)

Reply:
The fix for this is to set the default shell in the Dockerfile by using an ENV step to create the SHELL env vat.

Reply:
Completion for bash now configured as default.